cli_flags.go

```diff
@@ -299,6 +303,31 @@ func parseArgs() (*arguments, error) {
 				Destination: &args.site,
 				Sources:     cli.EnvVars("DD_HOST_PROFILING_SITE", "DD_SITE"),
 			},
+			&cli.StringFlag{
+				Name:        "extra-symbol-endpoints",
+				Usage:       "Extra endpoints to upload symbols to.",
+				Hidden:      true,
+				Destination: &extraSymbolEndpoints,
+				Sources:     cli.EnvVars("DD_HOST_PROFILING_ADDITIONAL_SYMBOL_ENDPOINTS"),
+				Action: func(_ context.Context, _ *cli.Command, s string) error {
+					if s == "" {
+						return nil
+					}
+					err := json.Unmarshal([]byte(s), &args.extraSymbolEndpoints)
+					if err != nil {
+						return errors.New("invalid JSON format for extra symbol endpoints")
+					}
+					for _, e := range args.extraSymbolEndpoints {
+						if !(e.Site != "" && e.APIKey != "" && e.AppKey != "") {
+							return errors.New("all values of JSON object must be set, non-empty strings")
+						}
+						if !isAPPKeyValid(e.AppKey) {
+							return errors.New("APP key is not valid")
```

Comment:
Nit: Let's add more context to the errors, to make this easier to debug:

```suggestion
							return fmt.Errorf("error parsing DD_HOST_PROFILING_ADDITIONAL_SYMBOL_ENDPOINTS: app key for site %s is not valid", e.Site)
```